parser/parser.go

```diff
@@ -96,6 +95,22 @@ func (obj *TrivialURIParser) Parse() ([]interfaces.Iterator, error) {
 
 	if isGit(u) {
 		// TODO: for now, just assume it can only be a git iterator...
+		// Checking if commit hash exists at the end of the URL.
+		// examples of URLs of different hosts containing commit hashes:
+		// github: https://github.com/awslabs/yesiscan/commit/496d080bc7fe835511d7220f127e118d0881b792
+		// webrtc: https://webrtc.googlesource.com/src.git/+/c276aee4eda7b1a466b139838f20e790bd746309
```

Comment:
OK. -- But please add a comment specifying that this might need to be generalized in the future as we add more URL patterns.

Reply:
Fixed.